hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncTable.java

```diff
@@ -792,6 +793,12 @@ <S, R> CoprocessorServiceBuilder<S, R> coprocessorService(Function<RpcChannel, S
    * server does not offer partial results, it is still safe to use this, assuming you implement
    * your {@link PartialResultCoprocessorCallback#getNextCallable(Object, RegionInfo)} correctly.
    */
-  <S, R> CoprocessorServiceBuilder<S, R> coprocessorService(Function<RpcChannel, S> stubMaker,
-    ServiceCaller<S, R> callable, PartialResultCoprocessorCallback<S, R> callback);
+  default <S, R> CoprocessorServiceBuilder<S, R> coprocessorService(
+    Function<RpcChannel, S> stubMaker, ServiceCaller<S, R> callable,
+    PartialResultCoprocessorCallback<S, R> callback) {
+    LoggerFactory.getLogger(AsyncTable.class).warn(
```

Comment:
Is warn appropriate here? Should it be debug instead, so it doesn't flood clients? Should it be warn, but guarded such that it will only log once in the process lifetime?